common/djangoapps/third_party_auth/samlproviderconfig/tests/test_samlproviderconfig.py

```diff
@@ -96,6 +97,7 @@ def test_get_one_config_by_enterprise_uuid_found(self):
         assert results[0]['entity_id'] == SINGLE_PROVIDER_CONFIG['entity_id']
         assert results[0]['metadata_source'] == SINGLE_PROVIDER_CONFIG['metadata_source']
         assert response.data['results'][0]['country'] == SINGLE_PROVIDER_CONFIG['country']
+        assert re.match(r"test-slug-\d{4}", results[0]['display_name'])
```

Comment:
should we also add/update test cases to verify the pre save works as intended for both cases (1) when display_name is provided vs (2) default case if not?

Reply:
perhaps `test_create_one_config` is a good test to update?